### PR TITLE
[ENHANCEMENT] Explorer - don't reuse PanelPreview but directly the Timeseries chart

### PR DESCRIPTION
**Pull request by celian-garcia**
# Description

This PR aims to remove another dependency from explorer to dashboard : The ``PanelPreview``. Indeed it's been done like this to prevent from code duplication, but the benefit is not worth it, since we'll refactor the view anyway.

Doing so, I move the ``useSuggestedStepMs`` utils directly alongside with the ``useTimeRange`` as very close. 

# Screenshots

This screenshot simulates a after/before, but what will remain is only the first one. This is to see that this is not very different from the previous one.
image

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the
  following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `DOC`,`IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [x] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [x] Visual tests are stable and unlikely to be flaky.
  See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests)
  and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues
  include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

File: ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 The Perses Authors
```
Comment (celian-garcia) [inline ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx:1]: This is actually a rename of the ``PanelEditorForm`` component right on top.

File: ui/explore/package.json [hunk 2]
```diff
@@ -23,14 +23,13 @@
     "build:types": "tsc --project tsconfig.build.json",
     "type-check": "tsc --noEmit",
     "start": "concurrently -P \"npm:build:* -- {*}\" -- --watch",
-    "test": "cross-env TZ=UTC jest",
```
Comment (celian-garcia) [inline ui/explore/package.json:26]: Doing so many removes that it remains nothing tested :sunglasses:

File: ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      <ExploreToolbar exploreTitleComponent={exploreTitleComponent} />
+      <Grid container spacing={2}>
+        <Grid item xs={12}>
+          <Stack gap={1}>
+            <TimeSeriesQueryEditor onChange={setQueries} queries={queries} />
+          </Stack>
+        </Grid>
+        <Grid item xs={12}>
+          <Stack gap={1}>
+            <Typography variant="h4">Preview</Typography>
```
Comment (AntoineThebaud) [inline ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx:82]: Not directly linked to your change but imho this "Preview" title should be either change to "Result" or simply removed.

**Review by AntoineThebaud — APPROVED** [review]
LGTM! Just got a single display remark

File: ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx (hunk 3 above)
Comment (celian-garcia) [reply ui/explore/src/components/TimeSeriesExplorer/TimeSeriesExplorer.tsx:82]: Sure! For now, I'm not taking care that much of changing the display as I'm working in parallel on a complete redesign. So I won´t modify that if you agree with me.

Spoiler alert (work in progress)
image

**Review by Gladorme — APPROVED** [review]
lgtm

**Pull request merged**